### ISSUE TITLE
Disabling the calendar and its options

### ISSUE THREAD

**Issue by vvalentin-lucca** [LuccaSA/lucca-front#1184]
It should be :

- add a real disabled attribute to the calendar options outside min and max (and not just do it with a class);
- disable the navigation buttons when the months are out of scope;
- prevent the calendar from being opened from the keyboard when it is disabled.
